BrainPortal/config/routes.rb

```diff
@@ -312,6 +312,8 @@
         post :upload_file
       end
     end
+    resources :nh_messages,        :except => [ :edit, :show ] do
```

Comment:
Why do you put an empty block? You don't need this. Look at other examples in the file